sklearn/metrics/tests/test_common.py

```diff
@@ -369,11 +369,11 @@
 
 # No Sample weight support
 METRICS_WITHOUT_SAMPLE_WEIGHT = [
-    "confusion_matrix", # Left this one here because the tests in this file do
-                        # not work for confusion_matrix, as its output is a
-                        # matrix instead of a number. Testing of
-                        # confusion_matrix with sample_weight is in
-                        # test_classification.py
+    "confusion_matrix",  # Left this one here because the tests in this file do
+    # not work for confusion_matrix, as its output is a
+    # matrix instead of a number. Testing of
+    # confusion_matrix with sample_weight is in
+    # test_classification.py
     "median_absolute_error",
```

Comment:
Any reason why this had to be edited from how it was previously @wanjiku-carol ?